coref/logging_utils.py

```diff
@@ -1,46 +1,29 @@
 import logging
-import warnings
 from logging import Logger
-from typing import Optional, Any
+from typing import Any
 
-from config.logging import LoggingConfig
-
-
-def get_logging_level(verbosity: int = LoggingConfig.verbosity) -> int:
-    try:
-        verbosity_mapped: int = LoggingConfig.verbosity_mapping[verbosity]
-        return verbosity_mapped
-    except (KeyError, NameError):
-        # Since these are logging utilities, we do not want to be dependent
-        #   on logging that can happen to be not set up;
-        # Using the UserWarning instead
-        warnings.warn(
-            f"verbosity={verbosity} could not be mapped to any logging level; "
-            f"returning the DEBUG level",
-        )
-        return logging.DEBUG
+from config.logging import Logging
 
 
 def get_stream_logger(
-    name: str,
```

Comment:
Getting rid of the customizable name (by only getting it from the initial config) disallows us to have multiple loggers for different sub-packages, which allows for identifying better places where shit happens :D

Do we not want this functionality?

However, another view is that forcing them to write to the same file might be tricky :/

Reply:
In my point of view if we want more loggers, they should be all defined in config... I think config is the ultimate file that sets up everything